modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -67,7 +67,12 @@ import {
   getFieldAttributeType
 } from './helpers/feature-attributes';
 import {NodeIndexDocument} from './helpers/node-index-document';
-import {loadNestedTileset, loadTile3DContent} from './helpers/load-3d-tiles';
+import {
+  isTilesetType,
```

Comment:
I find the name confusing. `isTileset` makes some sense, but `isTilesetType` is a bit much.

How about
```typescript
getTilesetType(tile): 'tileset' | 'tile'

if (getTilesetType(tile) === 'tileset') 
```
`

Reply:
Done